skops/io/tests/test_persist.py

```diff
@@ -804,3 +804,75 @@ def test_loads_from_str():
     msg = "Can't load skops format from string, pass bytes"
     with pytest.raises(TypeError, match=msg):
         loads("this is a string")
+
+
+class _BoundMethodHolder:
+    """Used to test the ability to serialize and deserialize bound methods"""
+
+    def __init__(self, object_state: str):
+        self.object_state = (
+            object_state  # Initialize with some state to make sure state is persisted
+        )
+        self.chosen_function = (
+            np.log  # bind some method to this object, could be any persistable function
+        )
+
+    def bound_method(self, x):
+        return self.chosen_function(x)
+
+
+class TestPersistingBoundMethods:
+    @staticmethod
+    def assert_transformer_persisted_correctly(
+        loaded_transformer: FunctionTransformer,
+        original_transformer: FunctionTransformer,
+    ):
+        """Checks that a persisted and original transformer are equivalent, including the func passed to it
```

Comment:
Could you please set a length limit inside the docstring (say, 79 or 88)? I know we haven't specified this anywhere, but I think it helps with readability and having a consistent look. Same applies to the method below.